src/index.js

```diff
@@ -0,0 +1,131 @@
+const updateColorsAndEmojis = () => {
+  if (state.temp > 80) {
+    document.getElementById('degrees').style.color = 'red';
+    document.getElementById('emojis').innerText = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+  } else if (state.temp >= 70 && state.temp <= 79) {
+    document.getElementById('degrees').style.color = 'orange';
+    document.getElementById('emojis').innerText = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+  } else if (state.temp >= 60 && state.temp <= 69) {
+    document.getElementById('degrees').style.color = 'yellow';
+    document.getElementById('emojis').innerText = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+  } else if (state.temp > 50 && state.temp <= 59) {
+    document.getElementById('degrees').style.color = 'green';
+    document.getElementById('emojis').innerText = '🌲🌲';
+  } else if (state.temp < 59) {
+    document.getElementById('degrees').style.color = 'teal';
+    document.getElementById('emojis').innerText =
+      '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  }
+};
+// ------------- Wave 3 ----------------------
+const updateCity = () => {
+  let myTextInput = document.getElementById('search-bar');
+  let cityName = myTextInput.value;
+  let cityHeader = document.getElementById('city-name');
+  cityHeader.textContent = `For the lovely city of: ${cityName}`;
+  state.city = cityName;
+};
+// ------------- Wave 4 --------------------
+const findLatitudeAndLongitude = () => {
+  axios
+    .get('http://127.0.0.1:5000/location', {
+      params: {
+        q: state.city,
+        format: 'json',
+      },
+    })
+    .then((response) => {
+      console.log(response.data);
+      latitude = response.data[0].lat;
+      longitude = response.data[0].lon;
+      console.log('success in findLatitudeAndLongitude!', latitude, longitude);
+      getWeather(latitude, longitude);
+    })
+    .catch((error) => {
+      console.log('error in findLatitudeAndLongitude!', error);
+    });
+};
+// -------------- Wave 5 --------------------
+const getSkyChoice = () => {
+  const skyDropdown = document.querySelector('select');
+  const skyChoice = skyDropdown.value;
+  console.log('sky choice:', skyChoice);
+  state.sky = skyChoice;
+  if (state.sky === 'sunny') {
+    document.getElementById('sky-emojis').innerText = '☁️ ☁️ ☁️ ☀️ ☁️ ☁️';
+  } else if (state.sky === 'cloudy') {
+    document.getElementById('sky-emojis').innerText =
+      '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️';
+  } else if (state.sky === 'rainy') {
+    document.getElementById('sky-emojis').innerText = '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧';
+  } else if (state.sky === 'snowy') {
+    document.getElementById('sky-emojis').innerText = '🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨';
+  }
+};
+// getting temperature from latitude and longitude
+const getWeather = (latitude, longitude) => {
+  axios
+    .get('http://127.0.0.1:5000/weather', {
+      params: { lat: latitude, lon: longitude },
+    })
+    .then((response) => {
+      console.log('success in getWeather!', response.data.main.temp);
+      let kelvin = response.data.main.temp;
+      let fahrenheit = Math.round(((kelvin - 273.15) * 9) / 5 + 32);
```

Comment:
Consider putting the temperature conversion in a separate function. Also, these can be `const` instead.